.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "algorithms/reveal"]
```

Comment:
I don't think we need some of these files. I'll clean them up in another branch.